libs/embed/src/embed.ts

```diff
@@ -14,6 +14,8 @@ const IFRAME_ID = 'novu-iframe-element';
 class Novu {
   public clientId: string | unknown;
 
+  private backendUrl: string;
```

Comment:
`Error: src/embed.ts(17,11): error TS2564: Property 'backendUrl' has no initializer and is not definitely assigned in the constructor.`
Add `this.backendUrl = '';` in constructor